README.md

```diff
@@ -39,6 +39,39 @@ If you're looking to utilize the capabilities of Starknet contracts within the R
 
 3. **Feedback**: Your feedback is invaluable to us 🌟! If you encounter any issues or have game-changing suggestions, don't hesitate to reach out through our [Discord](https://discord.com/invite/PaCMRFdvWT) or our [Community Forum](https://community.nethermind.io/). Let's make something awesome together! 🤝
 
+### Troubleshooting
+
+Most issues with Starknet plugin or Remix itself are caused by connectivity problems (also resulting from restricted networks, web-proxies blocking certain content, etc.) or browser plugin interference. 
+- A user should first attempt to disable any browser components & addons which may impact the connectivity or Javascript execution. 
+- Some networks may restrict connectivity to certain sites or domains. Using a VPN conneciton may resolve problems observed on restricted networks.
+
+More specific potential error causes are also described in detail below. 
+
+1. **Problems when searching plugins in Remix UI**  
```

Comment:
can we make these dropdown 

https://gist.github.com/citrusui/07978f14b11adada364ff901e27c7f61

Reply:
Dropdowns added